app/pages/items/description/template.test.js

```diff
@@ -0,0 +1,58 @@
+import { componentTester } from '../../../test-utils/componentTester';
+import manifest from './manifest.json';
+
+const setup = {
+  template: {
+    path: 'pages/items/description/template.njk',
+  },
+};
+
+const context = {
+  ...manifest,
+  title: 'org1 orders',
+  backlinkHref: '/organisation/order-1',
+  // TODO: Change below to be more in line with real href
+  saveButtonHref: '#',
+};
+
+describe('description page', () => {
+  it('should render the description page title', componentTester(setup, (harness) => {
+    harness.request(context, ($) => {
+      const title = $('h1[data-test-id="description-page-title"]');
+      expect(title.length).toEqual(1);
+      expect(title.text().trim()).toEqual(context.title);
+    });
+  }));
+
+  it('should render the description page description', componentTester(setup, (harness) => {
+    harness.request(context, ($) => {
+      const description = $('h2[data-test-id="description-page-description"]');
+      expect(description.length).toEqual(1);
+      expect(description.text().trim()).toEqual(context.description);
+    });
+  }));
+
+  it('should render the textareaFieldNoCount component', componentTester(setup, (harness) => {
+    harness.request(context, ($) => {
+      const input = $('[data-test-id="question-description"] textarea');
+      const footerAdvice = $('[data-test-id="textarea-field-footer"] span');
+
+      expect(input.length).toEqual(1);
+      expect(input[0].attribs.id).toEqual(context.descriptionQuestion.question.id);
+      expect(input[0].attribs.name).toEqual(context.descriptionQuestion.question.id);
+      expect(input[0].attribs.rows.trim())
```

Comment:
why the `[0]` is the selector finding an array? should it just be finding the single element

Reply:
it doesn't work without `[0]`

Reply:
for the selector can you try this;
`const input = $('[data-test-id="question-description"]').find('textarea')`;

Reply:
it is syntactic sugar for what I have written in line 37

Reply:
I think it is fine as the length is still getting checked - splitting hairs here

Reply:
just looks confusing that's all. If something we expect to be a single element we should reflect that. The data-test-id should help us getting that single element